src/com/generic/main/Shipment.java

```diff
@@ -0,0 +1,48 @@
+package com.generic.main;
+
+public class Shipment {
+	private String shipmentID;
+	private String freight;
+	private float weight;
+	private int receiptDate; //Need to figure out the date format
```

Comment:
I think instructor wanted us to use DateTime for this datatype.
This would mean that our fields does not necessarily need to match the datatype of the JSON file
as long as when we save it back, it retains the original JSON data type.

If this is the case, I think shipmentID can be an int